scripts/build_python.sh

```diff
@@ -64,6 +64,10 @@ case "$OS_TYPE" in
         ;;
 esac
 
+if [ -n "$chip_support_webrtc_python_bindings" ]; then
+    extra_gn_args+=("--args=chip_support_webrtc_python_bindings=true")
```

Comment:
This will not work. Wait for the changes in here https://github.com/project-chip/connectedhomeip/pull/39578 and add:
```bash
gn_args+=("chip_support_webrtc_python_bindings=true")
```